test/queue/queueservice_test.js

```diff
@@ -164,4 +171,70 @@ suite('queue/QueueService', function() {
     debug("pending message count: %j", count);
     assert(typeof(count) === 'number', "Expected count as number!");
   });
+
+  test("deleteUnusedWorkerQueues (can delete queues)", async () => {
+    // 11 days into the future, so we'll delete all queues (yay)
+    let now = new Date(Date.now() + 11 * 24 * 60 * 60 * 1000);
+    let deleted = await queueService.deleteUnusedWorkerQueues(now);
+    // There should always be at least one queue, because test cases above
+    // will have created one we can delete...
+    assume(deleted).is.atleast(1);
```

Comment:
so this is checking that the value returned is at least 1, but does not check that the queue indeed was deleted.  Could we then try to get the meta data on the queue and ensure that throws an exception?